apollo-runtime-kotlin/src/iosMain/kotlin/com/apollographql/apollo/network/http/ApolloHttpNetworkTransport.kt

```diff
@@ -152,6 +152,7 @@ actual class ApolloHttpNetworkTransport(
       val postBody = operation.composeRequestBody(scalarTypeAdapters).toByteArray().toNSData()
       setHTTPMethod("POST")
       headers
+          .plus("Content-Type" to "application/json; charset=utf-8")
```

Comment:
Is this something we can test?

Reply:
I must admit I'm a bit lazy to add a test there. The only thing I can think of is to use a mock server and check the headers but that feels like a lot of code for something that most likely won't be changing too much?

Reply:
👌